__tests__/app.test.js

```diff
@@ -0,0 +1,41 @@
+const request = require('supertest')
+const app = require('../app')
+const db = require('../db/connection.js')
+const seed = require('../db/seeds/seed')
+const testData = require('../db/data/test-data')
+
+beforeEach(() => seed(testData))
+afterAll(() => db.end())
+
+
+describe("app", () => {
+    describe("GET /api/topics", () => {
+        test("should return an array of objects with properties of slug and description", () => {
+            return request(app).get('/api/topics').expect(200).then((res) => {
+                expect(res.body.topics[0]).toMatchObject({
+                    slug: expect.any(String),
+                    description: expect.any(String)
+                })
+                expect(typeof res.body.topics).toBe("object")
+            })
+
+        });
+        test("should return sorted data by slug ASC", () => {
```

Comment:
At the moment we don't need to worry too much about this test as we aren't specifically searching for sorted data :) - that will come into play a little later on.